verify-and-fix-supabase.sh

```
@@ -0,0 +1,364 @@
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+print_status() { echo -e "${GREEN}[✓]${NC} $1"; }
+print_error() { echo -e "${RED}[✗]${NC} $1"; }
+print_warning() { echo -e "${YELLOW}[!]${NC} $1"; }
+print_info() { echo -e "${BLUE}[i]${NC} $1"; }
+
+# Your Supabase project details
+SUPABASE_PROJECT_ID="mxtsdgkwzjzlttpotole"
+SUPABASE_URL="https://mxtsdgkwzjzlttpotole.supabase.co"
+SUPABASE_ANON_KEY="REDACTED"
```

Shellcheck (0.11.0)

[warning] SUPABASE_ANON_KEY appears unused. Verify use (or export if used externally).

(SC2034)